src/widgets/rail_widget.h

```diff
@@ -10,6 +10,15 @@
 #ifndef WIDGETS_RAIL_WIDGET_H
 #define WIDGETS_RAIL_WIDGET_H
 
+/** Global hotkeys modifiers for #RailToolbarWidgets */
+enum RailToolbarWidgetModifiers {
+	GHK_MOD_RAT_PREV_RAILTYPE = 9, ///< bit added to WID_RAT_AUTORAIL to select previous railtype
+	GHK_MOD_RAT_NEXT_RAILTYPE = 10, ///< bit added to WID_RAT_AUTORAIL to select next railtype
+
+	GHK_MODB_RAT_PREV_RAILTYPE = 1 << GHK_MOD_RAT_PREV_RAILTYPE, ///< bitmask for GHK_MOD_RAT_PREV_RAILTYPE
```

Comment:
I wanted them way out of `WID_RAT_*` enum values since originally I used them as `GHK_MODB_RAT_PREV_RAILTYPE | WID_RAT_AUTORAIL` in `Hotkey` definition.

I didn't want to put them into `RailToolbarWidgets` since this actions have no corresponding buttons in toolbar itself and are a bit more global.